letsencrypt-auto-source/pieces/pipstrap.py

```diff
@@ -43,7 +42,7 @@ def check_output(*popenargs, **kwargs):
                 cmd = popenargs[0]
             raise CalledProcessError(retcode, cmd)
         return output
-from sys import exit, version_info
+import sys
```

Comment:
Just in case you didn't know, it's perfectly possible to import some things deep from a module and other things shallow.

Reply:
Yes it was just to save a line, but if you think it is a problem, as for instance by exposing the sys module as part of this module could be a problem, I can stay with a deep import for the other members.

Reply:
I mentioned it just because it made you change lines of code that otherwise wouldn't have needed to change. As for "security", it makes no difference whether you expose `sys` directly or not, as you can get to it by following the `__module__` attribute of anything defined in `sys`.

Reply:
Thanks!